benchmarks/asv.conf.json

```diff
@@ -20,7 +20,7 @@
 
     // The Pythons you'd like to test against.  If not provided, defaults
     // to the current version of Python used to run `asv`.
-    "pythons": ["3.6"],
+
```

Comment:
@WarrenWeckesser this was probably an accident, but what do you think about removing it? Is it important somewhere (e.g. [Pauli's server](https://pv.github.io/scipy-bench/)) that Python 3.6 is always used for benchmarks?

Reply:
^ Indeed it was inadvertent to this PR. I had to do some fiddling to get benchmarks to run locally.

Reply:
Yeah, not as easy as it should be.

Reply:
@mdhaber I  don't know--I've never taken a close look at the benchmark code.